drag-and-drop/library/src/controller.tsx

```diff
@@ -189,4 +189,9 @@ export class DragAndDropController {
     // we only want JSON compliant values
     return JSON.parse(JSON.stringify(removeDndStateFromView(baseView)));
   }
+
+  public importView(view: View, pluginName = '') {
```

Comment:
I don't think we should take `pluginName` as a parameter in this flow. Since JSON content doesn't have enough semantic information about where an Asset came from, we can only assume it came from the plugins that are already loaded into the SDK.

Reply:
removed pluginName parameter